src/argilla/client/feedback/schemas/records.py

```diff
@@ -226,9 +224,24 @@ class Config:
 
 
 class RemoteFeedbackRecord(FeedbackRecord):
+    """Schema for the records of a `FeedbackDataset` remotely (in Argilla).
```

Comment:
```suggestion
    """Schema for the records of a `RemoteFeedbackDataset`.
```

Maybe this is better and more clear as we're mentioning it later in the docstring.